nesc/whip6/platforms/parts/mcu/cc26xx/timers/HalEventCountPrv.nc

```diff
@@ -110,9 +111,19 @@ implementation {
         uint64_t ov_value;
         uint32_t cnt_value;
         atomic {
-            cnt_value = getTimerValue(which);
-            if (cnt_value < lastRead[getCntIdx(which)])
-                overflows[getCntIdx(which)] += MAX_COUNT;
+            cnt_value = getTimerValue(which) & (MAX_COUNT - 1);
+            if (which & TIMER_A) {
+                if (call ChannelAInterrupt.getPending() |
```

Comment:
Why is this condition different than in the original algorithm?

Reply:
how is it different?

Reply:
Why the bitwise or rather than just using getPending()?

Reply:
will talk about it at the meeting